src/Regex4.java

```diff
@@ -0,0 +1,20 @@
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Regex4 {
+    public static void main(String[] args) {
+
+
+        String regexToTest = "app=edi_adapter_splitter wingtipsTrace=225debfbe6e5fac7 poiFileName=Integration_test_" +
+                "Contract INFO  LogUtils - POI file name: [Integration_test_Contract], total number of orders succes" +
+                "sfully processed: [2]";
+        Pattern pattern = Pattern.compile("(?<=orders)(\\D*)(\\d)");
```

Comment:
to make the regex more robust, especially if the number of orders can be more than 9, you should use \\d+, you could simplify the regex to just orders[^\\d]*(\\d+), which will look for the word "orders" followed by any non-digit characters, and then one or more digits